string class
geometry_msgs/PoseStamped[] PoseStamped